webapi/src/test/scala/org/knora/webapi/slice/admin/domain/model/KnoraUserGroupSpec.scala

```diff
@@ -7,15 +7,15 @@ package dsp.valueobjects
 
 import zio.test.*
 
-import dsp.valueobjects.Group.*
+import org.knora.webapi.slice.admin.domain.model.*
 
 /**
  * This spec is used to test the [[Group]] value objects creation.
  */
-object GroupSpec extends ZIOSpecDefault {
+object KnoraUserGroupSpec extends ZIOSpecDefault {
   private val validDescription = Seq(V2.StringLiteralV2(value = "Valid group description", language = Some("en")))
   private val invalidDescription = Seq(
-    V2.StringLiteralV2(value = "Invalid group description \r", language = Some("en"))
+    V2.StringLiteralV2(value = "", language = Some("en"))
```

Comment:
Since `toSparqlEncodedString` is apparently no longer necessary, only empty strings are considered invalid.